HighAvailabilityEncodingStreaming/deploy.ps1

```diff
@@ -73,7 +73,7 @@ foreach ($functionName in $createdFunctionNames)
     Compress-Archive -Path $publishPath -DestinationPath $functionZipFilePath -Force
 
     Write-Host "Publishing Azure Function:" $functionName["fullName"].Value
-    Publish-AzWebApp -ResourceGroupName $ResourceGroupName -Name $functionName["fullName"].Value -ArchivePath $functionZipFilePath -Force
+    Publish-AzWebApp -Timeout 300000 -ResourceGroupName $ResourceGroupName -Name $functionName["fullName"].Value -ArchivePath $functionZipFilePath -Force
```

Comment:
I never had an issue with timeout, but I always used nearby data center. Timeout parameter was introduced in March 2021, it was not available in my environment, I had to install latest version of Az.Websites to get this to work. I think it is a little bit risky to add this parameter like this. It would probably be better to check version of Az.Websites and use it only when we know that it is going to work. Also, we could just add comment for now that suggests to add this parameter in case of failure.